app/models/reward.rb

```diff
@@ -10,4 +10,20 @@ class Reward < ApplicationRecord
   has_many :orders, dependent: :nullify
   has_many :category_rewards, dependent: :destroy
   has_many :categories, through: :category_rewards
+
+  def self.import(file)
+    ActiveRecord::Base.transaction do
+      CSV.foreach(file.path, headers: true) do |row|
+        reward_hash = row.to_hash
+        unless reward_hash['slug'].nil?
```

Comment:
I would avoid using `unless` in my opinion it makes code harder to read.

Also I am not 100% sure, but probably we should also cover the case, when there is no record with a matching slug ( as I get it it is like import from csv, but if we have it in db update it instead of updating)

```suggestion
        if reward_hash['slug'].present?
```

Reply:
when there is a slug in csv, but no in db, then i initialize reward, and then save new one.
```
reward = Reward.find_or_initialize_by(slug: reward_hash['slug'])
reward.save!
```

Reply:
You are right -  my bad - for some reason I misunderstood this statement -.-